src/builtins/printf.rs

```diff
@@ -438,7 +444,7 @@ impl<'a, 'b> builtin_printf_state_t<'a, 'b> {
                     let mut continue_looking_for_flags = true;
                     while continue_looking_for_flags {
                         match f.char_at(0) {
-                            'I' | '\'' => {
+                            '\'' => {
```

Comment:
I don't really know this code but yeah this looks like it would only change behavior on %I which is not documented, so looks good